tsl/test/expected/cagg_on_cagg_integer_dist_ht.out

```diff
@@ -376,11 +387,13 @@ SELECT * FROM :CAGG_NAME_2TH_LEVEL ORDER BY bucket;
 
 -- DROP the 2TH level CAGG don't affect others
 DROP MATERIALIZED VIEW :CAGG_NAME_2TH_LEVEL;
-psql:include/cagg_on_cagg_common.sql:199: NOTICE:  drop cascades to table _timescaledb_internal._hyper_3_3_chunk
+psql:include/cagg_on_cagg_common.sql:201: NOTICE:  drop cascades to table _timescaledb_internal._hyper_3_3_chunk
```

Comment:
I would actually disable notices here since they are not important for this test and will just generate false mismatches in the future.

Reply:
Fixed!